### PR TITLE
user can change password with token within 1 hour with link

### PR DESCRIPTION
**Pull request by norskemarcus**
added design to reset password page and validation of input

added validation

removed reset_password_token and expires from headerKeys

added some layout to edit password page

changed school to location in room search

### CODE REVIEW

File: client/src/routes/reset-password/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+			if (!response.ok) {
+				const result = await response.json();
+				toast.error(`Fejl: ${result.data}`);
+				return;
+			}
+
+			toast.success('Kodeordet er opdateret');
+			setTimeout(() => {
+				goto('/');
+			}, 2000);
```
Comment (tommyhellejensen) [inline client/src/routes/reset-password/+page.svelte:57]: Skal der være setTimeout her?

File: client/src/routes/reset-password/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+		} catch (error) {
+			console.error(error);
+			toast.error('Noget gik galt ved netværksanmodningen.');
+		}
+	}
+</script>
+
+<svelte:head>
+	<title>Reset password</title>
```
Comment (tomm8231) [inline client/src/routes/reset-password/+page.svelte:66]: Skal det ikke være på dansk?

File: client/src/routes/reset-password/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+						<button class="btn btn-primary" type="submit" style="width: 50%;">Gem</button>
+					</div>
+				</form>
+				<div class="alert alert-warning mt-5" role="alert">
+					Lav et stærkt kodeord med min. 6 tegn, skal indeholde bogstaver, et tal og et tegn.
+				</div>
+			</div>
+		</div>
+	</div>
+</main> -->
```
Comment (tommyhellejensen) [inline client/src/routes/reset-password/+page.svelte]: Slet?

File: server/routers/authRouter.js [hunk 4]
```diff
@@ -39,22 +40,62 @@ router.post('/auth/signup', adminCheck, async (req, res) => {
...
+//       hashed_password,
+//       role_id,
+//     })
+//
+//     res.status(200).send({ data: response })
+//   } else {
+//     res.status(400).send({ data: 'User was not created: user already exists' })
+//   }
+// })
+
```
Comment (tommyhellejensen) [inline server/routers/authRouter.js]: Slet?

**Review by tommyhellejensen — CHANGES_REQUESTED** [review]
Se mine kommentarer

File: client/src/routes/reset-password/+page.svelte (hunk 1 above)
Comment (norskemarcus) [reply client/src/routes/reset-password/+page.svelte:57]: Den skal være der, ellers ser man ikke toasten

**Review by tommyhellejensen — APPROVED** [review]
Godt arbejde!

**Pull request merged**